my-videos/src/components/MyVideos/index.js

```diff
@@ -0,0 +1,82 @@
+import React, { useEffect, useState } from 'react';
+import PropTypes from 'prop-types';
+import { forEach } from 'lodash';
+import './MyVideos.css';
+
+export default function MyVideos({ projectsUrl }) {
+  const [isLoaded, setIsLoaded] = useState(false);
+  const map = {
+    'square' : 'http://placekitten.com/g/100/100',
+    'landscape' : 'http://placekitten.com/g/250/150',
+    'vertical' : 'http://placekitten.com/g/100/200',
+  }
+
+  useEffect(async () => {
+    await fetch(projectsUrl)
+      .then(response => response.json())
+      .then(response => {
+        fetchedData = response.payload.projects;
+        setIsLoaded(true);
+      });
+
+    return fetchedData;
+  }, [isLoaded]);
+
+  return (
+    <div className="wrap">
+      {!isLoaded && (
+        <p>Please hold, while we load your videos...</p>
+      )}
+      {isLoaded && forEach(fetchedData, (project) => {
+        const {
+          aspect_ratio : aspectRatio,
+          presentation,
+          okey,
+          metadata,
+          links
+        } = project;
+
+        const thumbnailUrl = links.cover_image || map[aspectRatio]
+
+        if(aspectRatio == 'square') {
```

Comment:
from above -- using a const would allow us to do 

```suggestion
        if(aspectRatio === SQUARE_ASPECT) {
```

to reduce chance of typos